documentation/presentation-questions.md

```diff
@@ -4,10 +4,20 @@
 ## [Aleks] Matching Process/Prototype:
 Can you explain how mentors and mentees will be matched? Will you use a specific algorithm or something more generic (match based on which attributes overlap the most)?
 
+- The process of how mentors and mentees will be matched will be decided by the skills and skill levels that both the mentor and mentee possess. The mentor will then be able to view all of the mentees that have matched based on their skills and their level in those skills. The ultimate decision will be up to the mentor to accept the mentee.
```

Comment:
Can you also mention how timezones and common availability will be considered in the matching process?